gpsdio_segment/segment.py

```diff
@@ -1,11 +1,14 @@
-from __future__ import division
+from __future__ import print_function, division
 
-from itertools import chain
+from collections import namedtuple
 import logging
-from gpsdio_segment.state import SegmentState 
 
 logger = logging.getLogger(__file__)
-# logger.setLevel(logging.DEBUG)
+logger.setLevel(logging.INFO)
+
+
+SegmentState = namedtuple('SegmentState', 
+    ['id', 'ssvid', 'first_msg', 'last_msg', 'msg_count', 'noise', 'closed', 'opaque'])
```

Comment:
What do you call  `opaque`?

Reply:
The `opaque` field is used for the `pipe-segment` side of things to pass values through
the segmentizer that are "opaque" to the segmentizer in the sense that it can't (or shouldn't in this case) look into, just pass them along.  This was used for some of the identity information, but I just pulled that out, so I could remove this -- debating that at the moment.

Reply:
I needed to do some adjusting to the fields in `Segment` (adding first and last message)
for each day), so while I was doing that I removed the `opaque` field.